ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,275 @@
+                    ->arrayNode('link')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\LinkFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('api_client')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\ApiClientFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('api_client_collection')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\ApiClientCollectionFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('datatable_translation')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\DatatableTranslationFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('trash_item')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\TrashItemFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('trash_item_collection')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\TrashItemCollectionFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('translation')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\TranslationFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('widget_collection')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\WidgetCollectionFacade')->end()
+                        ->end()
+                    ->end()
+                    ->arrayNode('widget')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->scalarNode('facade')->defaultValue('OpenOrchestra\ApiBundle\Facade\WidgetFacade')->end()
+                        ->end()
+                    ->end()
+                ->end()
+            ->end()
+        ->end();
```

Comment:
Maybe to get this more readable, you could use a protected method to add a facade to the tree and use it iterating an array containing the facade names and classes.